github/resource_github_branch_protection.go

```diff
@@ -205,7 +220,7 @@ func resourceGithubBranchProtectionRead(d *schema.ResourceData, meta interface{}
 	}
 
 	if err := flattenAndSetRequiredPullRequestReviews(d, githubProtection); err != nil {
-		return fmt.Errorf("Error setting required_pull_request_reviews: %v", err)
+		return fmt.Errorf("Error setting required_pull_reuest_reviews: %v", err)
```

Comment:
typo: `required_pull_request_reviews`

Reply:
Hmm looks like that was an old diff, I see it fixed here 96803823fb5504ffe293504b88c8fa3c98b3fd78